docs/book/src/user/quick-start.md

```diff
@@ -604,12 +604,14 @@ export VSPHERE_RESOURCE_POOL="*/Resources"
 export VSPHERE_FOLDER="vm"
 # The VM template to use for your VMs
 export VSPHERE_TEMPLATE="ubuntu-1804-kube-v1.17.3"
-# The VM template to use for the HAProxy load balancer of the management cluster
-export VSPHERE_HAPROXY_TEMPLATE="capv-haproxy-v0.6.0-rc.2"
 # The public ssh authorized key on all machines
 export VSPHERE_SSH_AUTHORIZED_KEY="ssh-rsa AAAAB3N..."
-
-clusterctl init --infrastructure vsphere
```

Comment:
Q: Are we sure we want to drop this instruction for vSphere (the users can miss this step if not instructed to do so....)

Reply:
never mind, just noticed this is in the generate template section

Reply:
This instruction is already a part of the variable setup for vSphere. Hence removed it from here.
https://github.com/kubernetes-sigs/cluster-api/blame/main/docs/book/src/user/quick-start.md#L330

Reply:
Page didn't refresh, missed your second comment. :+1: